src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -187,6 +188,16 @@ public async Task<ActionResult> CreateApiReview(
 
             if (review != null)
             {
+                // Update package type if provided from controller parameter
+                if (!string.IsNullOrEmpty(packageType) && Enum.TryParse<PackageType>(packageType, true, out var parsedPackageType))
+                {
+                    if (review.PackageType != parsedPackageType)
+                    {
+                        review.PackageType = parsedPackageType;
+                        review = await _reviewManager.UpdateReviewAsync(review);
+                    }
```

Comment:
Hm... not sure this is the right logic. This says "if package type is supplied and different from what's saved, update it."

I think the actual logic should be closer to:
- If the current review has no packageType and packageType is supplied, update it
- If review has a packageType that matches the supplied packageType, do nothing
- If review.packageType is not null and a packageType is supplied, and they DO NOT agree, that would be an error. A management plane cannot suddenly switch to a data-plane, or vice versa.